Makefile

```diff
@@ -123,12 +128,15 @@ simphony-openfoam:
 	@echo "Simphony OpenFoam plugin installed"
 
 simphony-jyu-lb:
-	pip install --upgrade git+https://github.com/simphony/simphony-jyulb.git@0.1.1#egg=jyu_engine
+	rm -Rf src/simphony-jyulb
+	git clone --branch 0.1.2 --depth 1 https://github.com/simphony/simphony-jyulb.git src/simphony-jyulb
+	git clone --branch master https://github.com/simphony/JYU-LB.git src/simphony-jyulb/JYU-LB
```

Comment:
commonly one can pass the path of the header files when calling setup.py. something like.

```python 
python setup.py build_ext -I <include directory> -L <library directory> install
```